dags.yaml

```diff
@@ -12,6 +12,8 @@ bqetl_error_aggregates:
     retries: 1
     retry_delay: 20m
     depends_on_past: false
+  tags:
+    - impact/tier1
```

Comment:
We need to document what this means somewhere. I'd say either mana or the telemetry-airflow repository (where we might want to tag other things)?

Reply:
can we also add the proposed source repo tag

Reply:
Hey @wlach 👋 ,

Added ability to specify tags via the `bqetl dag create` command and added some additional info to the linked cookbook. I agree though long term we should probably have a separate page for it that we can just link to (this is just a temp measure).

Reply:
@haroldwoo , Will bring this up during the next platform meeting to make sure everyone would be okay with adding that tag before doing it since the agreement so far was to only include this tag.

Reply:
@wlach I added a doc: https://github.com/mozilla/bigquery-etl/blob/feat/airflow-tags/docs/reference/airflow_tags.md

Please let me know your thoughts.